plugins/woocommerce/tests/php/includes/emails/class-wc-email-customer-pos-refunded-order-test.php

```diff
@@ -37,7 +37,7 @@ public function test_order_item_totals_adds_formatted_cash_change_due_amount_to_
 
 		// Then cash payment change due amount is set and formatted correctly.
 		$this->assertArrayHasKey( 'cash_payment_change_due_amount', $totals );
-		$this->assertEquals( wc_price( '5.00', array( 'currency' => $order->get_currency() ) ), $totals['cash_payment_change_due_amount']['value'] );
+		$this->assertEquals( wc_price( '0', array( 'currency' => $order->get_currency() ) ), $totals['cash_payment_change_due_amount']['value'] );
```

Comment:
This probably needs to check that we _don't_ add the change for the refund emails... even if it's non-zero.

Reply:
Test cases updated in https://github.com/woocommerce/woocommerce/pull/58012/commits/ae9001a162e6f85ddbb599cdb37486761b45711d.